### ISSUE TITLE
Documentation: Update the Release Guide to Document the Proposed Workflow During the RC1 Cycle

### ISSUE THREAD

**Issue by jprestop** [dtcenter/METplus#2322]
## Describe the Task ##
From the [METplus 5.1.0 Coordinated Release Retrospective](https://docs.google.com/document/d/1_cfRYFYuYqazet793OSojZ-MPE5-CwEq0h-E-jIdnbM/edit), we discussed allowing a month for testing, if possible.  However, we want to ensure there is not downtime in development cycle and how we likely need to revise workflow.  We did this for MET for the MET-11.1.0 release, where I believe:

- we made main_v11.1 our main branch instead of leaving it main_v11.0
- we created branches from main_v11.1 instead of develop for any development tasks needed to main_v11.1
- we created branches from develop for any new development for main_v12.0 (but not for main_v11.1)
- after the main_v11.1 release, we merged main_v11.1 into develop

@JohnHalleyGotway can you please review the above notes for accuracy when you get a chance?

The task is to update the release guide, likely for creating an RC1 release.

### Time Estimate ###
< 1 day of work

### Sub-Issues ###
Consider breaking the task down into sub-issues.
- [ ] *Add a checkbox for each sub-issue here.*

### Relevant Deadlines ###
See Project Cycle

### Funding Source ###
Ask Tara before starting on this work.

### Assignee ###
- [ ] Select **engineer(s)** or **no engineer** required
- [ ] Select **scientist(s)** or **no scientist** required

### Labels ###
- [x] Select **component(s)**
- [x] Select **priority**
- [x] Select **requestor(s)**

### Projects and Milestone ###
- [x] Select **Repository** and/or **Organization** level **Project(s)** or add **alert: NEED CYCLE ASSIGNMENT** label
- [x] Select **Milestone** as the next official version or **Future Versions**

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [x] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdataio](https://github.com/dtcenter/METdataio/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)

## Task Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [ ] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>_<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Add any new Python packages to the [METplus Components Python Requirements](https://metplus.readthedocs.io/en/develop/Users_Guide/overview.html#metplus-components-python-requirements) table.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)** and **Development** issues
Select: **Repository** level development cycle **Project** for the next official release
Select: **Milestone** as the next official version
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.

**Comment by JohnHalleyGotway** [comment]
This issue is stale and is super-ceded by METplus #2754 which creates the main_vX.Y branch when the rc1 development release is created.
